core/src/main/java/io/substrait/relation/ProtoRelConverter.java

```diff
@@ -77,8 +82,11 @@ public Rel from(io.substrait.proto.Rel rel) {
       case CROSS -> {
         return newCross(rel.getCross());
       }
+      case EXTENSION_SINGLE -> {
+        return newExtensionSingle(rel.getExtensionSingle());
+      }
       default -> {
-        // TODO: add support for EXTENSION_SINGLE, EXTENSION_MULTI, EXTENSION_LEAF
+        // TODO: add support for EXTENSION_MULTI, EXTENSION_LEAF
```

Comment:
I see some methods implemented for these other cases below (EXTENSION_MULTI, EXTENSION_LEAF), but are they not completely implemented?